llvm/lib/Target/PowerPC/PPC.td

```diff
@@ -435,8 +435,7 @@ def ProcessorFeatures {
      FeatureP9Vector,
      FeaturePPCPreRASched,
      FeaturePPCPostRASched,
-     FeatureISA3_0,
-     FeaturePredictableSelectIsExpensive
+     FeatureISA3_0
```

Comment:
Might be a bit of a bigger refactor than you're looking to do here, but the AArch64 backend is a much better model for this: ISA features, and Tune features are separated out into parallel lists.

Reply:
Agree, RISCV is doing that separation as well